lib/dry/system/container.rb

```diff
@@ -328,15 +329,21 @@ def load_paths!(*dirs)
         #     auto_register!('lib/core')
         #
         #     # with a dir and a custom registration block
-        #     auto_register!('lib/core') do |component|
-        #       # custom way of initializing a component
+        #     auto_register!('lib/core') do |config|
+        #       config.instance do |component|
+        #         custom way of initializing a component
+        #       end
+        #
+        #       config.exclude do |component|
+        #         custom way for excluding components
```

Comment:
Would probably be best to read `# return true for components to exclude from auto-registration`